Makefile

```diff
@@ -52,47 +50,23 @@ xc: dependencies test test-long REBUILD
 clean:
 	rm -rf release doc/whitepaper.aux doc/whitepaper.log doc/whitepaper.pdf
 
-# test runs the short tests for Sia, and aims to always take less than 2
-# seconds.
+# 3 commands and a variable are available for testing Sia packages. 'pkgs'
+# indicates which packages should be tested, and defaults to all the packages
+# with test files. Using './...' as default breaks compatibility with the cover
+# command. 'test' runs short tests that should last no more than a few seconds,
+# 'test-long' runs more thorough tests which should not last more than a few
+# minutes.
+pkgs = ./api ./compatibility ./crypto ./encoding ./modules/consensus            \
+	./modules/gateway ./modules/host ./modules/hostdb ./modules/miner           \
+	./modules/renter ./modules/transactionpool ./modules/wallet ./siad ./siag   \
+	./types
```

Comment:
you could use `find` to list these: `find . -type d | grep -v 'git\|release\|doc'`

unfortunately if you have a testing directory or other untracked directory in your Sia folder, it's gonna cause problems. So I guess we're stuck manually specifying them.